core/src/test/java/io/grpc/StatusTest.java

```diff
@@ -49,4 +52,25 @@ public void verifyExceptionMessage() {
     assertEquals("CANCELLED: This is a test",
         Status.CANCELLED.withDescription("This is a test").asException().getMessage());
   }
+
+  @Test
+  public void impossibleCodeValue() {
+    assertEquals(Code.UNKNOWN, Status.fromCodeValue(-1).getCode());
+  }
+
+  @Test
+  public void sameCauseReturnsSelf() {
+    assertSame(Status.CANCELLED, Status.CANCELLED.withCause(null));
+  }
+
+  @Test
+  public void sameDescriptionReturnsSelf() {
+    assertSame(Status.CANCELLED, Status.CANCELLED.withDescription(null));
+    assertSame(Status.CANCELLED, Status.CANCELLED.augmentDescription(null));
+  }
+
+  @Test
+  public void useObjectHashCode() {
+    assertEquals(Status.CANCELLED.hashCode(), System.identityHashCode(Status.CANCELLED));
```

Comment:
Seems the only reason this test exists is for the test coverage, which seems bad, does JaCoCo have something similar to `// COV_NF_XXX` in Google3, so that we can specify the code that shouldn't be counted for test coverage?